code/modules/mob/living/carbon/human/species/machine.dm

```diff
@@ -1,6 +1,7 @@
 /datum/species/machine
 	name = "Machine"
 	name_plural = "Machines"
+	max_age = 150
```

Comment:
The current in-game year is always 2567, and IPCs were first built in 2510. While IPCs may be able to live to 150, in-universe no extant IPC can be older than ~50. This is the only species young enough that there's actually a discrepancy between the oldest possible member and the maximum age of the species.